packages/malte-common/src/UserList.ts

```diff
@@ -0,0 +1,9 @@
+export interface User {
```

Comment:
Should really use the already existing interface for `User` (in ` packages/malte-common/src/oauth/Github.ts`), unless you have a really good reason for not doing so.